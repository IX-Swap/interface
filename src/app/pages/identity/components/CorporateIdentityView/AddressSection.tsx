import React from 'react'
import { Grid } from '@mui/material'
import { LabelledValue } from 'components/LabelledValue'
import { CorporateIdentity } from 'app/pages/identity/types/forms'

export interface AddressSectionProps {
  address: CorporateIdentity['companyAddress']
}

export const AddressSection = ({ address }: AddressSectionProps) => {
  if (address === undefined) {
    return null
  }

  const { line1, line2, city, postalCode, state, country } = address

  return (
    <Grid
      item
      sx={{
        display: 'grid',
        gridTemplateColumns: { sx: '1fr', sm: '1fr 1fr' }
      }}
      container
      spacing={5}
    >
      <Grid item container direction={'column'} spacing={5}>
        <Grid item>
          <LabelledValue isRedesigned value={line1} label='Line 1' />
        </Grid>
        <Grid item>
          <LabelledValue isRedesigned value={city} label='City' />
        </Grid>
        <Grid item>
          <LabelledValue isRedesigned value={country} label='Country' />
        </Grid>
      </Grid>

      <Grid item container direction={'column'} spacing={5}>
        <Grid item>
          <LabelledValue isRedesigned value={line2} label='Line 2' />
        </Grid>
        <Grid item>
          <LabelledValue isRedesigned value={state} label='State' />
        </Grid>
        <Grid item>
          <LabelledValue isRedesigned value={postalCode} label='Postal Code' />
        </Grid>
      </Grid>
    </Grid>
  )
}
