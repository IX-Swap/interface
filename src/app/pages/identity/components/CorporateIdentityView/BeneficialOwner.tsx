import React from 'react'
import { Grid } from '@mui/material'
import { LabelledValue } from 'components/LabelledValue'
import { Personnel } from 'app/pages/identity/types/forms'
import { FormSectionHeader } from 'ui/FormSectionHeader/FormSectionHeader'
import { FieldContainer } from 'ui/FieldContainer/FieldContainer'

import { ProofDocuments } from 'app/pages/identity/components/CorporateIdentityView/ProofDocuments'

export interface BeneficialOwnerProps {
  data: Personnel
  title: string
}

export const BeneficialOwner = ({ data, title }: BeneficialOwnerProps) => {
  const { fullName, percentageShareholding, documents } = data

  return (
    <FieldContainer>
      <Grid container direction={'column'} spacing={5}>
        <Grid item>
          <FormSectionHeader title={title} />
        </Grid>

        <Grid
          item
          container
          spacing={5}
          sx={{
            display: 'grid',
            gridTemplateColumns: { sx: '1fr', sm: '1fr 1fr' }
          }}
        >
          <Grid item container direction={'column'} spacing={5}>
            <Grid item>
              <LabelledValue isRedesigned value={fullName} label='Full Name' />
            </Grid>
          </Grid>
          <Grid item container direction={'column'} spacing={5}>
            <Grid item>
              <LabelledValue
                isRedesigned
                value={
                  percentageShareholding !== undefined
                    ? `${percentageShareholding}%`
                    : 'â€“'
                }
                label='Percentage Shareholding'
              />
            </Grid>
          </Grid>
        </Grid>

        <ProofDocuments documents={documents} />
      </Grid>
    </FieldContainer>
  )
}
