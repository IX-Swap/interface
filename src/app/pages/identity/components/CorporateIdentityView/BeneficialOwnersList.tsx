import { Grid } from '@material-ui/core'
import { BeneficialOwner } from 'app/pages/identity/components/CorporateIdentityView/BeneficialOwner'
import React from 'react'
import { Personnel } from 'app/pages/identity/types/forms'

export interface BeneficialOwnersListProps {
  personnel: Personnel[]
  showDocumentHeader?: boolean
  documentsTitle?: string
}

export const BeneficialOwnersList = ({
  personnel,
  showDocumentHeader = false,
  documentsTitle = 'Documents'
}: BeneficialOwnersListProps) => {
  if (personnel.length < 1) {
    return null
  }

  return (
    <Grid container direction='column' spacing={6}>
      {personnel.map((personnel, i) => (
        <Grid item key={i}>
          <BeneficialOwner
            personnel={personnel}
            showDocumentHeader={showDocumentHeader}
            documentsTitle={documentsTitle}
          />
        </Grid>
      ))}
    </Grid>
  )
}
