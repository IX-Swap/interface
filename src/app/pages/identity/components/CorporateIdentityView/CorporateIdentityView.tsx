import React from 'react'
import { Grid } from '@mui/material'
import { FormSectionHeader } from 'app/pages/identity/components/FormSectionHeader'
import { PersonnelList } from 'app/pages/identity/components/CorporateIdentityView/PersonnelList'
import { BeneficialOwnersList } from 'app/pages/identity/components/CorporateIdentityView/BeneficialOwnersList'
import { CorporateAddress } from 'app/pages/identity/components/CorporateIdentityView/CorporateAddress'
import { CorporateInfo } from 'app/pages/identity/components/CorporateIdentityView/CorporateInfo'
// import { CountryTaxDeclaration } from 'app/pages/identity/components/CountryTaxDeclarations/CountryTaxDeclaration'
import { CorporateIdentity } from 'app/pages/identity/types/forms'
import { FieldContainer } from 'app/pages/identity/components/FieldContainer/FieldContainer'
import { OwnershipStructure } from 'app/pages/identity/components/CorporateIdentityView/OwnershipStructure'
import { DirectorList } from 'app/pages/identity/components/CorporateIdentityView/DirectorList'
// import { InvestorDeclarationView } from 'app/pages/identity/components/CorporateIdentityView/InvestorDeclarationView'
// import { OptInView } from 'app/pages/identity/components/CorporateIdentityView/OptInView'
// import { DocumentsView } from 'app/pages/identity/components/CorporateIdentityView/DocumentsView'
// import { InstitutionalInvestorDeclarationView } from 'app/pages/identity/components/CorporateIdentityView/InstitutionalInvestorDeclarationView'

export interface CorporateIdentityViewProps {
  data: CorporateIdentity
  isIssuer?: boolean
}

export const CorporateIdentityView = ({ data }: CorporateIdentityViewProps) => {
  return (
    <Grid container spacing={2} direction='column'>
      <Grid item>
        <FieldContainer>
          <Grid container direction='column' spacing={5}>
            <Grid item>
              <FormSectionHeader
                title='Review Responses'
                hasBottomBorder={true}
              />
            </Grid>

            <Grid item>
              <FormSectionHeader title='Corporate Information' />
            </Grid>

            <CorporateInfo data={data} />
          </Grid>
        </FieldContainer>
      </Grid>

      <Grid item>
        <FieldContainer>
          <Grid container direction='column' spacing={5}>
            <Grid item>
              <FormSectionHeader title='Ownership Structure Layers' />
            </Grid>

            <OwnershipStructure data={data} />
          </Grid>
        </FieldContainer>
      </Grid>

      <Grid item>
        <CorporateAddress data={data} />
      </Grid>

      <Grid item>
        <PersonnelList personnel={data.representatives ?? []} />
      </Grid>

      <Grid item>
        <DirectorList data={data} />
      </Grid>

      <Grid item>
        <BeneficialOwnersList data={data} />
      </Grid>

      {/* <Grid item>
        <CountryTaxDeclaration taxResidencies={data.taxResidencies} />
      </Grid>

      <Grid item>
        <InvestorDeclarationView data={data} />
      </Grid>

      <Grid item>
        <OptInView data={data} />
      </Grid>

      <Grid item>
        <InstitutionalInvestorDeclarationView data={data} />
      </Grid>

      <Grid item>
        <DocumentsView data={data.documents} />
      </Grid> */}
    </Grid>
  )
}
