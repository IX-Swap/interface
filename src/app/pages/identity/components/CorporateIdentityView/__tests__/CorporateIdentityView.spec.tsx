import { CorporateIdentityView } from 'app/pages/identity/components/CorporateIdentityView/CorporateIdentityView'
import { history } from 'config/history'
import React from 'react'
import { render, cleanup } from 'test-utils'
import { corporate } from '__fixtures__/authorizer'

window.URL.revokeObjectURL = jest.fn()

describe('CorporateIdentityView', () => {
  beforeEach(() => {
    history.push('/', { identityId: '123' })
  })

  afterEach(async () => {
    await cleanup()
    jest.clearAllMocks()
  })

  it('renders without errors', () => {
    render(<CorporateIdentityView data={corporate} />)
  })
})
