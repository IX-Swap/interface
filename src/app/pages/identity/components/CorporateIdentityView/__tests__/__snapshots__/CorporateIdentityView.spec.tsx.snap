// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CorporateIdentityView should match snapshot 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-6 MuiGrid-direction-xs-column css-167wck0-MuiGrid-root"
  >
    <div
      class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
      style="padding-bottom: 0px;"
    >
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 css-1syvn0i-MuiPaper-root"
      >
        <div
          class="MuiBox-root css-5vb4lz"
        >
          <h5
            class="MuiTypography-root MuiTypography-h5 css-14eqi03-MuiTypography-root"
          >
            Overview
          </h5>
        </div>
        <div
          class="MuiBox-root css-1xhj18k"
        >
          <div
            class="MuiBox-root css-8qb8m4"
          >
            <div
              class="MuiAvatar-root MuiAvatar-square MuiAvatar-colorDefault css-19j9e2s-MuiAvatar-root"
              style="width: 128px; height: 128px; border-radius: initial; max-width: initial;"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiAvatar-fallback css-1ql7le6-MuiSvgIcon-root-MuiAvatar-fallback"
                data-testid="PersonIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
                />
              </svg>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-direction-xs-column css-32q0xd-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
              style="padding-bottom: 0px;"
            >
              <div
                class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 css-zow5z4-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 css-n7cpkr-MuiGrid-root"
                >
                  <div
                    class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column css-80wlp8-MuiGrid-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1b4l2pg-MuiTypography-root"
                      style="font-weight: 700; width: 100%; color: rgb(59, 66, 81);"
                    >
                      Company Name
                    </p>
                    <div
                      class="MuiBox-root css-1gbh5gc"
                    />
                    <p
                      class="MuiTypography-root MuiTypography-body1 MuiTypography-alignLeft css-1ctkodk-MuiTypography-root"
                      style="font-weight: 400; opacity: 1; width: 100%; word-break: break-word;"
                    >
                      InvestaX
                    </p>
                  </div>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 css-n7cpkr-MuiGrid-root"
                >
                  <div
                    class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column css-80wlp8-MuiGrid-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1b4l2pg-MuiTypography-root"
                      style="font-weight: 700; width: 100%; color: rgb(59, 66, 81);"
                    >
                      Company Registration Number/UEN
                    </p>
                    <div
                      class="MuiBox-root css-1gbh5gc"
                    />
                    <p
                      class="MuiTypography-root MuiTypography-body1 MuiTypography-alignLeft css-1ctkodk-MuiTypography-root"
                      style="font-weight: 400; opacity: 1; width: 100%; word-break: break-word;"
                    >
                      123456
                    </p>
                  </div>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 css-n7cpkr-MuiGrid-root"
                >
                  <div
                    class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column css-80wlp8-MuiGrid-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1b4l2pg-MuiTypography-root"
                      style="font-weight: 700; width: 100%; color: rgb(59, 66, 81);"
                    >
                      Status of Risk Report
                    </p>
                    <div
                      class="MuiBox-root css-1gbh5gc"
                    />
                    <p
                      class="MuiTypography-root MuiTypography-body1 MuiTypography-alignLeft css-1ctkodk-MuiTypography-root"
                      style="font-weight: 400; opacity: 1; width: 100%; word-break: break-word;"
                    >
                       
                    </p>
                  </div>
                  <p
                    class="MuiTypography-root MuiTypography-body1 makeStyles-authStatus-1 makeStyles-unknown-8 makeStyles-authStatusNewTheme-2 makeStyles-unknownNewTheme-9 css-1b4l2pg-MuiTypography-root"
                  >
                    UNKNOWN
                  </p>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
              style="padding-top: 0px;"
            >
              <div
                class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 css-zow5z4-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 css-n7cpkr-MuiGrid-root"
                >
                  <div
                    class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column css-80wlp8-MuiGrid-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1b4l2pg-MuiTypography-root"
                      style="font-weight: 700; width: 100%; color: rgb(59, 66, 81);"
                    >
                      Country of Incorporation
                    </p>
                    <div
                      class="MuiBox-root css-1gbh5gc"
                    />
                    <p
                      class="MuiTypography-root MuiTypography-body1 MuiTypography-alignLeft css-1ctkodk-MuiTypography-root"
                      style="font-weight: 400; opacity: 1; width: 100%; word-break: break-word;"
                    >
                      Russian Federation
                    </p>
                  </div>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 css-n7cpkr-MuiGrid-root"
                >
                  <div
                    class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column fs-exclude css-80wlp8-MuiGrid-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1b4l2pg-MuiTypography-root"
                      style="font-weight: 700; width: 100%; color: rgb(59, 66, 81);"
                    >
                      Legal Entity
                    </p>
                    <div
                      class="MuiBox-root css-1gbh5gc"
                    />
                    <p
                      class="MuiTypography-root MuiTypography-body1 MuiTypography-alignLeft css-1ctkodk-MuiTypography-root"
                      style="font-weight: 400; opacity: 1; width: 100%; word-break: break-word;"
                    >
                      other
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
    >
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 css-1syvn0i-MuiPaper-root"
      >
        <div
          class="MuiBox-root css-5vb4lz"
        >
          <h5
            class="MuiTypography-root MuiTypography-h5 css-14eqi03-MuiTypography-root"
          >
            Address
          </h5>
        </div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 css-zow5z4-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-15fcrl2-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column css-80wlp8-MuiGrid-root"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-1b4l2pg-MuiTypography-root"
                style="font-weight: 700; width: 100%; color: rgb(59, 66, 81);"
              >
                Address for Correspondence
              </p>
              <div
                class="MuiBox-root css-1gbh5gc"
              />
              <p
                class="MuiTypography-root MuiTypography-body1 MuiTypography-alignLeft css-1ctkodk-MuiTypography-root"
                style="font-weight: 400; opacity: 1; width: 100%; word-break: break-word;"
              >
                Omsk, Russian Federation, Siberia, Address line 1, Address line 2, 123456
              </p>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-15fcrl2-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column css-80wlp8-MuiGrid-root"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-1b4l2pg-MuiTypography-root"
                style="font-weight: 700; width: 100%; color: rgb(59, 66, 81);"
              >
                Registered Address
              </p>
              <div
                class="MuiBox-root css-1gbh5gc"
              />
              <p
                class="MuiTypography-root MuiTypography-body1 MuiTypography-alignLeft css-1ctkodk-MuiTypography-root"
                style="font-weight: 400; opacity: 1; width: 100%; word-break: break-word;"
              >
                Omsk, Russian Federation, Siberia, Address line 1, Address line 2, 123456
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
    >
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 css-1syvn0i-MuiPaper-root"
      >
        <div
          class="MuiBox-root css-5vb4lz"
        >
          <h5
            class="MuiTypography-root MuiTypography-h5 css-14eqi03-MuiTypography-root"
          >
            Company Authorized Personnel
          </h5>
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
    >
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 css-1syvn0i-MuiPaper-root"
      >
        <div
          class="MuiBox-root css-5vb4lz"
        >
          <h5
            class="MuiTypography-root MuiTypography-h5 css-14eqi03-MuiTypography-root"
          >
            Directors/Partners/People with Executive Authority
          </h5>
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
      style="padding-bottom: 0px; padding-top: 0px;"
    >
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 css-1syvn0i-MuiPaper-root"
      >
        <div
          class="MuiBox-root css-5vb4lz"
        >
          <h5
            class="MuiTypography-root MuiTypography-h5 css-14eqi03-MuiTypography-root"
          >
            Beneficial Owners Information
          </h5>
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
    >
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 css-1syvn0i-MuiPaper-root"
      >
        <div
          class="MuiBox-root css-5vb4lz"
        >
          <h5
            class="MuiTypography-root MuiTypography-h5 css-14eqi03-MuiTypography-root"
          >
            Investor Declaration
          </h5>
        </div>
        <div
          class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4 css-480o17-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
            >
              <h6
                class="MuiTypography-root MuiTypography-subtitle1 css-1b4l2pg-MuiTypography-root"
              >
                I declare that I am a corporate "Accredited Investor"
              </h6>
            </div>
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-spacing-xs-2 css-1tw2i9j-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
              >
                <div
                  class="MuiBox-root css-1yp4ln"
                >
                  <div
                    class="MuiBox-root css-0"
                    style="opacity: 0.5;"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1oaawcc-MuiSvgIcon-root"
                      data-testid="declarations-list-item-unchecked"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                      />
                    </svg>
                  </div>
                  <div
                    class="MuiBox-root css-d0uhtl"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1b4l2pg-MuiTypography-root"
                      data-testid="declarations-list-item-label"
                    >
                      An entity or corporation with net assets exceeding SGD 10 million or its equivalent in foreign currency; or
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
              >
                <div
                  class="MuiBox-root css-1yp4ln"
                >
                  <div
                    class="MuiBox-root css-0"
                    style="opacity: 0.5;"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1oaawcc-MuiSvgIcon-root"
                      data-testid="declarations-list-item-unchecked"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                      />
                    </svg>
                  </div>
                  <div
                    class="MuiBox-root css-d0uhtl"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1b4l2pg-MuiTypography-root"
                      data-testid="declarations-list-item-label"
                    >
                      The trustee of a trust the subject matter of which exceeds SGD 10 million or its equivalent in foreign currency; or
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
              >
                <div
                  class="MuiBox-root css-1yp4ln"
                >
                  <div
                    class="MuiBox-root css-0"
                    style="opacity: 0.5;"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1oaawcc-MuiSvgIcon-root"
                      data-testid="declarations-list-item-unchecked"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                      />
                    </svg>
                  </div>
                  <div
                    class="MuiBox-root css-d0uhtl"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1b4l2pg-MuiTypography-root"
                      data-testid="declarations-list-item-label"
                    >
                      A corporation where all the shareholders are Accredited Investors; or
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
              >
                <div
                  class="MuiBox-root css-1yp4ln"
                >
                  <div
                    class="MuiBox-root css-0"
                    style="opacity: 0.5;"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1oaawcc-MuiSvgIcon-root"
                      data-testid="declarations-list-item-unchecked"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                      />
                    </svg>
                  </div>
                  <div
                    class="MuiBox-root css-d0uhtl"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1b4l2pg-MuiTypography-root"
                      data-testid="declarations-list-item-label"
                    >
                      A partnership (other than a limited liability partnership) where all the partners are Accredited Investors; or
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
              >
                <div
                  class="MuiBox-root css-1yp4ln"
                >
                  <div
                    class="MuiBox-root css-0"
                    style="opacity: 0.5;"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1oaawcc-MuiSvgIcon-root"
                      data-testid="declarations-list-item-unchecked"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                      />
                    </svg>
                  </div>
                  <div
                    class="MuiBox-root css-d0uhtl"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1b4l2pg-MuiTypography-root"
                      data-testid="declarations-list-item-label"
                    >
                      A trust where all the beneficiaries are Accredited Investors; or
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
              >
                <div
                  class="MuiBox-root css-1yp4ln"
                >
                  <div
                    class="MuiBox-root css-0"
                    style="opacity: 0.5;"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1oaawcc-MuiSvgIcon-root"
                      data-testid="declarations-list-item-unchecked"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                      />
                    </svg>
                  </div>
                  <div
                    class="MuiBox-root css-d0uhtl"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-1b4l2pg-MuiTypography-root"
                      data-testid="declarations-list-item-label"
                    >
                      A trust where all the settlors are Accredited Investors and have reserved to themselves all powers of investment and asset management functions under the trust, and have reserved to themselves the power to revoke the trust.
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
          >
            <div
              class="MuiBox-root css-h8m1q6"
            >
              <div
                class="makeStyles-formHeader-28 MuiBox-root css-1xn0qx2"
              >
                <h5
                  class="MuiTypography-root MuiTypography-h5 css-14eqi03-MuiTypography-root"
                >
                  Opt-In Requirement
                </h5>
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4 css-480o17-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
            >
              <h6
                class="MuiTypography-root MuiTypography-subtitle1 css-1b4l2pg-MuiTypography-root"
              >
                I confirm to be treated as an “Accredited Investor” by InvestaX
              </h6>
            </div>
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-spacing-xs-2 css-1tw2i9j-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
              >
                <div
                  class="MuiBox-root css-1yp4ln"
                >
                  <div
                    class="MuiBox-root css-0"
                    style="opacity: 0.5;"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1oaawcc-MuiSvgIcon-root"
                      data-testid="declarations-list-item-unchecked"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                      />
                    </svg>
                  </div>
                  <div
                    class="MuiBox-root css-d0uhtl"
                  >
                    I have been informed of and understand my right to
                     
                    opt out
                     of the Accredited Investors status
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
    >
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 css-1syvn0i-MuiPaper-root"
      >
        <div
          class="MuiBox-root css-5vb4lz"
        >
          <h5
            class="MuiTypography-root MuiTypography-h5 css-14eqi03-MuiTypography-root"
          >
            Tax Declaration
          </h5>
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
    >
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 css-1syvn0i-MuiPaper-root"
      >
        <div
          class="MuiBox-root css-5vb4lz"
        >
          <h5
            class="MuiTypography-root MuiTypography-h5 css-14eqi03-MuiTypography-root"
          >
            Company Documents
          </h5>
        </div>
        <div
          class="MuiTableContainer-root css-rorn0c-MuiTableContainer-root"
        >
          <table
            class="MuiTable-root css-7mlt34-MuiTable-root"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-wrap-xs-nowrap css-1ledfka-MuiGrid-root"
            >
              <div
                class="MuiBox-root css-1ezo613"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 css-1b4l2pg-MuiTypography-root"
                  style="font-weight: 500;"
                >
                  File
                </p>
              </div>
              <div
                class="MuiBox-root css-19afw6t"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 css-1b4l2pg-MuiTypography-root"
                  style="font-weight: 500;"
                >
                  Type
                </p>
              </div>
              <div
                class="MuiBox-root css-19afw6t"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 css-1b4l2pg-MuiTypography-root"
                  style="font-weight: 500;"
                >
                  Uploaded At
                </p>
              </div>
              <div
                class="MuiBox-root css-19afw6t"
              />
            </div>
            <tbody
              class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
            />
          </table>
        </div>
      </div>
    </div>
  </div>
</div>
`;
